### ISSUE TITLE
Misc improvements to the web site:

### ISSUE THREAD

**Issue by nthiery** [OpenDreamKit/OpenDreamKit.github.io#18]
- [X] Shorten the main intro
- [X] list of mailing lists on the contact page
- [x] Integrate twitter feed?
- [x] Links to the task/deliverable/milestones issues on github
- [x] List of the different sites and people:
  - [x] with links to the presentations during the Kickoff
  - [x] with the roles of the participants, taken from the proposal
  - [x] with pictures
  - [ ] with recruited people and local close collaborators
- [x] List of meetings / meetings web pages? we already have a calendar and announces
- [x] Export the calendar as ICAL.
- [x] List of software?
- [x] Document in the [README.md](/OpenDreamKit/OpenDreamKit.github.io/) how to build the site with Jekyll
- [ ] How do we do analytics?
- [ ] Choose and host fonts locally
- [ ] Add four entry points:
  - "For mathematicians"
  - "For developers"
  - "For participants"
  - "For reviewers"
  
  For the first two, see https://github.com/OpenDreamKit/OpenDreamKit/issues/157
  For participants, see https://github.com/OpenDreamKit/OpenDreamKit/
  For reviewers, see https://github.com/OpenDreamKit/OpenDreamKit/issues/173
- [x] Update acknowledge page
- [x] Add a [page](http://opendreamkit.org/activities/) for listing talks and other ODK's activities
- [x] Mention somewhere (end of list of partners?) that the project was labelled by the [Free and Open Source Software Work Group of the Systematic Cluster](http://www.systematic-paris-region.org/en/get-info-topics/free-and-open-source-software).
- [x] Enable slides with reveal.js
- [x] Rename "activities" to "blog" (also in the main README). Maybe fuse "news", "activities", and "job openings" using a tags?
- [ ] Encourage more blog posts (Luca volunteered to lead this)
- [x] Remove Acknowledge from the main page (maybe also Edit this page if easy)
- [ ] Add tooltips on Task / Deliverable / Timeline to explain what those are.

**Comment by fangohr** [comment]
Regarding listing _talks, workshops, posters,_ etc -- we could have a new item on the left called 'activities' in which we list each 'activity' briefly, and -- if it deserves more detail -- it is a link to a separate page? I believe gathering as much data as possible from the beginning will pay off. 
(For my doctoral training centre, we have a news/blog page where I try to list all we do; this pays off at reporting times when you wouldn't remember otherwise what you have done, and is also a good online representation of the activities: http://ngcm.soton.ac.uk/blog/)

**Comment by nthiery** [comment]
On Mon, Dec 07, 2015 at 01:35:05PM -0800, Hans Fangohr wrote:

>    Regarding listing talks, workshops, posters, etc -- we could have a new
>    item on the left called 'activities' in which we list each 'activity'
>    briefly, and -- if it deserves more detail -- it is a link to a
>    separate page?

I like the term "activities". The simplest is probably to use Jekyll's
blog support (as we already do for our News section). Adding a new
post is just adding a short file, so it's fine to create one even for
just a brief link).

I'll try to add such a section sometime soon, unless someone beats me
to it!

>    I believe gathering as much data as possible from the beginning
>    will pay off.  (For my doctoral training centre, we have a
>    news/blog page where I try to list all we do; this pays off at
>    reporting times when you wouldn't remember otherwise what you
>    have done, and is also a good online representation of the
>    activities: [1]http://ngcm.soton.ac.uk/blog/)

+1!!!

Cheers,

##                 Nicolas

Nicolas M. Thiéry "Isil" nthiery@users.sf.net
http://Nicolas.Thiery.name/

**Comment by nthiery** [comment]
Hi @minrk!
For our website and elsewhere in our infrastructure, it would be helpful to have the data from our proposal (deliverables, ...) as a yaml file. When you generated our github issues from this data, you must have written a parser for the "final.pdata" file, right? Where could I find it? Then I could just use it to get a yaml dump.
Thanks in advance!

**Comment by minrk** [comment]
@nthiery I used [this ruby script](https://gist.github.com/47ada32480cc0eaba962c13a656eb6dc) to parse final.pdata.

**Comment by nthiery** [comment]
@defeo, @VivianePons, @kohlhase: I just pushed a script to convert the metadata from the proposal about wp's, tasks, deliverables into yaml, so that we could easily load it into Python/Jekyll/... for misc purposes.
- https://github.com/OpenDreamKit/OpenDreamKit/blob/master/Proposal/final.pdata.yaml
- https://github.com/OpenDreamKit/OpenDreamKit/blob/master/bin/pdata-latex-to-yaml
  So if we copy the output file into the website repo, we can now easily use it to generate pieces of the website. A first nice little utility would be to have taskref and delivref tags.

@kohlhase: you probably want to look over the changes I made to the proposal style file, as I made little changes to the pdata format for more consistency between tasks and deliverables + a bug fix.

Cheers

**Comment by defeo** [comment]
Are you sure the script is not buggy? I have a doubt about these lines https://github.com/OpenDreamKit/OpenDreamKit/blob/master/Proposal/final.pdata.yaml#L18-L29 and similar ones.

**Comment by nthiery** [comment]
Yeah; if we actually need the information on those lines, the script will need to be made less stupid.
In final.pdata, the label of a deliverable is of the form wp-label@deliv-label (same for tasks). I found it more convenient to split such labels in two chunks. And the splitting is a bit too agressive right now :-)

**Comment by VivianePons** [comment]
Many of the boxes have been answered by the new site.

In particular: there is a working ical link.
